src/Core/Worker.php

```diff
@@ -650,7 +650,8 @@ private static function killStaleWorkers()
 				$argv[0] = basename($argv[0]);
 
 				// How long is the process already running?
-				$duration = (time() - strtotime($entry["executed"])) / 60;
+				// For some weird reasons we cannot use "time()" here. It doesn't seem to be in UTC.
```

Comment:
`time()` doesn't have a timezone, it's supposed to be the Universal Unix Time, What was the issue?

Reply:
For some strange reasons the return value of ```time()``` is different to the return value of ```strtotime(DateTimeFormat::utcNow())``` - although both should return the UTC timestamp. Since we store the "executed" date with ```DateTimeFormat::utcNow()``` it is the best to compare it with the same return value.

Reply:
Weird.

Reply:
I did some testing and found the reason. ```strtotime``` respects the local timezone if no timezone text value is given. This means that ```strtotime($entry["executed"])``` doesn't return the correct timestamp when you aren't living in the UK.

I'm currently unsure how to handle this the best. Of we can set the timezone to "utc" for the workerqueue. But there are some places in the code that aren't called from the worker who use strtotime as well.